Cinema system/Cinema system/Controllers/OrdersController.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using CinemaSystem.Interfaces;
+using CinemaSystem.Models;
+
+namespace CinemaSystem.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class OrdersController : ControllerBase
+    {
+
+        private readonly IOrderService _orderService;
+
+        private const string ErrorOfUserNonexistence = "This user does not exists.";
+        private const string ErrorOfOrderNonexistence = "This order does not exists.";
+        private const string ErrorOfSeanceNonexistence = "This seance does not exists.";
+        private const string ErrorOfSeatBooked = "This seat does not exists or has already booked";
+
+        public OrdersController(IOrderService orderService)
+        {
+            _orderService = orderService;
+        }
+
+        [HttpGet("{userId}")]
+        public IActionResult GetOrders(int userId)
+        {
+            if (!_orderService.CheckUserExists(userId))
+            {
+                return NotFound(new Response(ErrorOfUserNonexistence));
+            }
+
+            var orders = _orderService.GetOrders(userId);
+
+            return Ok(new GetResponse<IEnumerable<OrderView>>(orders));
+        }
+
+        [HttpGet("{userId}/user/{orderId}")]
+        public IActionResult GetOrder(int userId, int orderId)
+        {
+            if (!_orderService.CheckUserExists(userId))
```

Comment:
Don't you have this method in user service? Whose responsibility is that?

Reply:
Yes, as I understand it, the authenticator responsible for this. I just decided that I still need to check when getting user id from url.